lib/browser/lighthouse.js

```diff
@@ -34,6 +34,7 @@
 
 const assert = require('assertive');
 const isEmpty = require('lodash/isEmpty');
+const deepMerge = require('deepmerge');
```

Comment:
does this do something for us that `lodash/merge` doesn't?

Reply:
thanks @dbushong . let me look into it